pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/security/JWTConverter.java

```diff
@@ -182,6 +183,15 @@ private DDPAuth convertJWT(String jwt, boolean useCachedIfAvailable) {
                     try {
                         DecodedJWT validToken = verifyDDPToken(jwt, jwkProvider);
                         String ddpUserGuid = validToken.getClaim(Auth0Constants.DDP_USER_ID_CLAIM).asString();
+                        //handle if auth0 user_id was set rather than user guid
+                        if (ddpUserGuid != null && ddpUserGuid.startsWith("auth0")) {
```

Comment:
As of now experimental for prion only.
will be setting this in auth0 action for prion